src/rust/engine/process_execution/src/remote_cache.rs

```diff
@@ -71,7 +72,14 @@ impl CommandRunner {
     concurrency_limit: usize,
   ) -> Result<Self, String> {
     let tls_client_config = if action_cache_address.starts_with("https://") {
-      Some(grpc_util::create_tls_config(root_ca_certs)?)
+      Some(
+        grpc_util::TlsConfig {
```

Comment:
Similar comment to before. This code is duplicated and probably should be in a constructor on `TlsConfig`.